frb_example/pure_dart/rust/src/api.rs

```diff
@@ -725,3 +725,111 @@ impl ConcatenateWith {
         Ok(())
     }
 }
+
+pub enum CustomError {
```

Comment:
As discussed earlier, maybe we need to use `thiserror` in order to attach a backtrace to it?

Reply:
My idea was to always pass a backtrace but I think your idea is better because it uses the already existing type conversion.